release_notes/47-release-notes.adoc

```diff
@@ -0,0 +1,348 @@
++
+For example, `0.300s`, `-5400s`, or `9900s`. The previously valid time units of `ns`, `us`, `µs`, `ms`, `m`, and `h` are no longer supported.
+
+7. The *Vulnerability Management* -> *Dashboard* view is deprecated and is planned to be removed in a future release. You can use the *Workload CVEs*, *Exception Management*, *Platform CVEs*, and *Node CVEs* views as alternatives.
+--
+
+[id="removed-features_{context}"]
+=== Removed features
+
+The following section provides information about the removed features:
+
+* Scanner V4 support for openSUSE Leap 15.0 and 15.1 has been removed.
+
+[id="bug-fixes_{context}"]
+== Bug fixes in version 4.7.0
+
+*Release date*: 3 March 2025
+
+//ROX-27588
+* Before this update, the queries did not take into account the new run state changes in downloadable reports, as the reports remained in `DOWNLOAD` state even after downloading. This issue occurred because the backend introduced a new `DELIVERED` state when you download the reports through the user interface (UI). With this update, the queries for recognizing the transition from `DOWNLOAD` to `DELIVERED` have been updated.
+
+//ROX-26763
+* Before this update, the logs repeatedly displayed the following error message:
++
+[source,text]
+----
+ [Throttled] Could not determine network namespace: No such file or directory
+----
++
+This issue occurred because the scraper that reads the process details in `procfs` reported errors when it encountered zombie processes, even though this was not necessarily a true error. Workloads that frequently generated zombie processes were more likely to trigger this message. 
++
+With this update, the system now specifically recognizes zombie processes and adjusts the message level to a less strict classification. However, the system can still trigger an error if the detection of zombie processes exceeds a certain threshold, helping to identify faulty workloads.
+
+//ROX-25638
+* Before this update, the Central logs were not rotated, which caused the log file for {product-title-short} to grow indefinitely and eventually take up the entire node memory. This issue occurred because `/var/log/stackrox` was mounted by using an `emptyDir` volume, which does not persist across pod restarts and has no built-in log rotation. 
++
+With this update, logs are deleted and the `emptyDir` volume is recreated when you restart the Central pod. A log size limit has been introduced to prevent excessive memory usage and to ensure that the Central logs do not overload the node.
+
+//ROX-27414 and ROX-27439
+* Before this update, Central filtered the `storage.IndexReport` message, which represents the state of RPM packages on a {op-system-first} node, due to the missing timestamps. 
++
+This caused issues in {product-title-short} 4.6.0 and 4.6.1 where identical `IndexReport` messages sent at different times were treated as duplicates even though they could return different sets of vulnerabilities.
```

Comment:
Limiting the blast radius of this issue: Affected were only the users (of 4.6.0 and 4.6.1) who decided to test the technical-preview feature of RHCOS Node Scanning with Scanner V4.